src/RelayPricer.ts

```diff
@@ -0,0 +1,42 @@
+import CoinBase from './api/CoinBase';
+import { SourceApi } from './types/RelayPricer';
+
+export default class RelayPricer {
+    private readonly sourceApi: SourceApi;
+
+    constructor(sourceApi: SourceApi | string) {
```

Comment:
**NIT**:
I'd simplify the constructor to receive argument of type `SourceApi`. IMHO the creation of the `SourceAPI` object should be done outside this class.

Reply:
We could that change, but I was thinking more about having the implementations on the rif-relay-client, and not exporting the SourceApi

Reply:
updated

Reply:
Did you mean that you wouldn't export the SourceApi outside the rif-relay-client library? If so, we could build and export a factory that performs what you were doing in the constructor before.

Reply:
I updated it, I had a conversion with Andres regarding it and he mention something like that, but to keep simple I decided to not do the implementation, yet.